part-3/pk-sim-compounds-definition-and-work-flow.md

```diff
@@ -278,60 +274,64 @@ Protein Binding Partners
+| In vitro active transport (vesicular assay) - Michaelis-Menten | A transporter mediated transfer of a compound across a cellular membrane is described as saturable process following Michaelis-Menten type kinetics. The corresponding kcat value is calculated from a specific Vmax normalized to the amount of transporter as input value which was determined in a vesicular transport assay.                                                                                                                                                  | • In vitro Vmax / transporter [nmol/min/pmol transfer]<br>• Km [µmol/l] |
 
 **Renal Clearances**
 
 | Process Type                         | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Necessary Input Parameter                   |
 | ------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------- |
-| Kidney Plasma Clearance              | <p>When choosing this process type experimental values for kidney plasma clearance can be used as input parameters. The specific clearance used in the simulation is then automatically calculated on the basis of the following parameters:<br>• Body weight<br>• Volume (kidney)<br>• Hematocrit<br>• Blood flow rate (kidney)<br>• Fraction unbound in plasma<br>• Plasma clearance<br>The default value for fraction unbound (experiment) is 0.5 in PK-Sim®. Please change this parameter if you have experimental data for the fraction unbound.</p> | Plasma clearance; Fraction unbound (plasma) |
```

Comment:
In PK-Sim, the "fraction unbound" is "Fraction unbound (experment)". But this is a **plasma** clearnace process, this should always be plasma clearance?

Reply:
Don't understand the question

Reply:
let's create another issue for this.